### ISSUE TITLE
Other hidden class define on other libraries

### ISSUE THREAD

**Issue by mathieu-stennier** [rpocklin/angular-scroll-animate#15]
Dear developers,

I struggled with this problem so I wanted to share the issue I got when using your library (which is awesome, thanks!)

I used the class `hidden` like mentioned in your README. 

```
.hidden {
  visibility: hidden;
}
```

Problem was that this css class was already used in another library I was using in my project and had the following attribute:

```
.hidden {
  display: none;
}
```

the fact is that the javascript function you are using in your library to retrieve the position of the element is  `getBoundingClientRect()` which returns (0,0,0,0) for element having `display:none` as attributes. Bad luck. So I changed the name of the visibility class to:

```
.notvisible {
  visibility: hidden;
}
```

and it worked again.

Hope that will help someone 👍 

Cheers and good work,

Mathieu.

**Comment by chaos6631** [comment]
Thanks, ran into the same thing with bootstrap

**Comment by rpocklin** [comment]
Will sort this one out, thanks.

**Comment by ghost** [comment]
Ran into the same issue. 
I changed the class name as suggested and it solved the issues.

The directive is amazing!!!

**Comment by rpocklin** [comment]
This has been fixed in `v0.9.9` please take a look and see if that helps (read the changelog for direction).
